src/app/components/dashboard-apps-dock/dashboard-apps-dock.less

```diff
@@ -0,0 +1,304 @@
+@mno-dock-bg-color: @decorator-inverse-color;
+@mno-dock-app-title-color: white;
+
+@mno-dock-sub-menu-bg-color: @darkblue;
+@mno-dock-sub-menu-text-color: white;
+@mno-dock-sub-menu-icon-color: @green;
+
+@mno-dock-app-logo-square-width: 70px;
+@mno-dock-app-logo-minimized-square-width: 50px;
+
+
+#app-dock-container {
+  position: absolute;
+  left: 0;
+  top: 0;
+  right: 0;
+  text-align: center;
+}
+
+ul.dock {
+  position: relative;
+  display: inline-block;
+  background-color: @mno-dock-bg-color;
+  opacity: 1;
+  padding: 5px;
+  z-index: 999;
+  .border-bottom-radius(5px);
+  height: 90px;
+  .transition(height 0.3s);
+
+  & > li {
+    display: inline-block;
+    position: relative;
+    vertical-align: top;
+
+    .title {
+      position: absolute;
+      top: 0;
+      color: @mno-dock-app-title-color;
+      display: none;
+      width: 100%;
+      text-align: center;
+    }
+
+    img.app-logo {
+      outline: none;
+      cursor: pointer;
+      margin: 5px;
+      .square(@mno-dock-app-logo-square-width);
+      .transition(all 0.3s);
+      .border-top-radius(5px);
+      .border-bottom-radius(5px);
+      .transform-origin(50% 0%); // Origin from top
+    }
+
+    .add-app {
+      cursor: pointer;
+      margin: 5px;
+      .square(@mno-dock-app-logo-square-width);
+      .transition(all 0.3s);
+      .transform-origin(50% 0%); // Origin from top
+      border: 1px solid @mno-dock-add-app-border-color;
+      .border-top-radius(5px);
+      .border-bottom-radius(5px);
+
+      display: table;
+
+      a {
+        display: table-cell;
+        vertical-align: middle;
+        color: @mno-dock-add-app-text-color;
+      }
+    }
+  }
+
+  // Expand button
+  li.expand-menu {
+    border-left: 1px solid @mno-dock-expand-left-border-color;
+    height: 100%;
+
+    & > div {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      height: 100%;
+
+      a {
+        padding-left: 6px;
+        padding-right: 5px;
+        .transition(height 0.3s);
+        cursor: pointer;
+        color: @mno-dock-expand-arrow-color;
+
+        &:hover {
+          text-decoration: none;
+          color: @mno-dock-expand-arrow-color-hover;
+        }
+      }
+    }
+  }
+
+  //=======================================
+  // Submenu
+  //=======================================
+  .dock-menu {
+    position: absolute;
+    left: 50%;
+    top: 100%;
+    z-index: 1000;
+
+    // Temp / To Review
+    margin-top: 68px;
+    // display: none;
+
+    visibility:hidden;
+    opacity:0;
+    transition: visibility 0s linear 0.5s,opacity 0.5s linear;
+
+    div.dock-arrow {
+      position: relative;
+      right: 50%;
+    }
+
+    div.inner-dock-menu { // Inner div holding the menu
+      position: relative;
+      right: 50%;
+      padding: 5px 0px;
+      .border-top-radius(5px);
+      .border-bottom-radius(5px);
+      background-color: @bg-inverse-color;
+
+      .box-shadow(0 6px 12px rgba(0, 0, 0, 0.175));
```

Comment:
Create a less variable for the values of the box shadow.
- in varables.less:
```
@dock-blah-blah-shadow: 0 6px 12px rgba(0, 0, 0, 0.175)
```
- Use it like this:
```
.box-shadow(@dock-blah-blah-shadow);
```